app/src/main/java/com/evercheck/flagly/developeroptions/FeatureFlagHandlerPresenter.kt

```diff
@@ -28,19 +33,21 @@ class FeatureFlagHandlerPresenter @Inject constructor(
 
     override fun onViewReady() {
         view?.setup()
-        setupFeatureFlagValues()
+    }
+
+    override fun filterFeautreFlagsByName(query: String) {
```

Comment:
there was a little mistake in my comment is feature not Feautre, please change it

Reply:
Done